lib/internal/Magento/Framework/Filesystem/Glob.php

```diff
@@ -19,6 +19,16 @@ class Glob extends LaminasGlob
      */
     private static $cache = [];
 
+    /**
+     * Clear the cache.
```

Comment:
Consider adding the below documentation, you can update accordingly. Its just a suggestion.
```suggestion
     * Clear the static cache for glob patterns.
     * This method should be used primarily in testing environments
     * or long-running processes where file system changes occur
     * between glob() calls and fresh results are required.
```